src/main/java/service/impl/FileWriterCsvImpl.java

```diff
@@ -0,0 +1,18 @@
+package service.impl;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import service.FileWriterService;
+
+public class FileWriterCsvImpl implements FileWriterService {
+    @Override
+    public void fileWriter(String filePath) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(filePath))) {
+            CreateReportImpl createReport = new CreateReportImpl();
+            bufferedWriter.write(createReport.generateReport());
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/229492946-09864370-e9eb-4b48-a794-288d326db5ce.png)

You still creating report inside Write service(
**Make your classes simple, reusable and focused on a single problem.**
How are you going to use this WriteService to write to file New Year greeting for employees? Is it independent? 
![image](https://user-images.githubusercontent.com/20326858/229491955-c27f43df-b09c-480f-841a-9f722e22bdb9.png)